src/openeo_grass_gis_driver/jobs_job_id.py

```diff
@@ -80,10 +80,32 @@ def get(self, job_id):
 
                         # Store the updated job in the database
                         self.job_db[job_id] = job
-                else:
+                elif code == 400:
+                    # actinia response contains only status and message
                     if job.additional_info != job_info:
                         job.additional_info = job_info
+                        if job_info["status"] == "finished":
+                            job.status = "finished"
+                        if job_info["status"] == "error":
+                            job.status = "error"
+                        if job_info["status"] == "accepted":
+                            job.status = "queued"
+                        if job_info["status"] == "terminated":
+                            job.status = "canceled"
+                        if job_info["status"] == "running":
+                            job.status = "running"
+
+                        # Store the updated job in the database
                         self.job_db[job_id] = job
+                else:
+                    # other 4xx errors
```

Comment:
The comment is misleading as all other HTTP codes are handled here, including eg. internal server errors 5xx